http/vibe/http/server.d

```diff
@@ -624,15 +624,7 @@ final class HTTPServerSettings {
 
 		The default value is 60 seconds; set to Duration.zero to disable pings.
 	*/
-	Duration webSocketPingInterval;// = dur!"seconds"(60);
-
-	this()
-	@safe {
-		// need to use the contructor because the Ubuntu 13.10 GDC cannot CTFE dur()
```

Comment:
Heh, who would have guessed that I had made the same changes yesterday :) https://github.com/rejectedsoftware/vibe.d/commit/42fdf87aad30442837da20b9eeef75c149adcb14

I'll merge this though and just drop that branch.